connector/connect/client/jvm/pom.xml

```diff
@@ -33,6 +33,7 @@
   <properties>
     <sbt.project.name>connect-client-jvm</sbt.project.name>
     <guava.version>31.0.1-jre</guava.version>
+    <mima.version>1.1.0</mima.version>
```

Comment:
The latest version is 1.1.1

Reply:
Yes, there is a bug in 1.1.1, where the MiMa will not be able to check the class methods if the object is marked `private`. Thus I used the same one that our SBT build uses, which is 1.1.0.